<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportImagePreview</class>
 <widget class="QDialog" name="ExportImagePreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Preview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="buttonZoomOut">
       <property name="text">
        <string notr="true">-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonZoom100">
       <property name="text">
        <string notr="true">100%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonZoomIn">
       <property name="text">
        <string notr="true">+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelZoom">
       <property name="text">
        <string notr="true">Zoom: 123%</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scroll">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLabel" name="preview">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionZoomIn">
   <property name="text">
    <string notr="true">+</string>
   </property>
   <property name="iconText">
    <string notr="true">+</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string notr="true">-</string>
   </property>
   <property name="iconText">
    <string notr="true">-</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionZoom100">
   <property name="text">
    <string notr="true">100%</string>
   </property>
   <property name="iconText">
    <string notr="true">100%</string>
   </property>
   <property name="toolTip">
    <string>Zoom to 100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>buttonClose</tabstop>
  <tabstop>buttonZoomIn</tabstop>
  <tabstop>buttonZoom100</tabstop>
  <tabstop>buttonZoomOut</tabstop>
  <tabstop>scroll</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>ExportImagePreview</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>ExportImagePreview</receiver>
   <slot>onZoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>ExportImagePreview</receiver>
   <slot>onZoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom100</sender>
   <signal>triggered()</signal>
   <receiver>ExportImagePreview</receiver>
   <slot>onZoom100()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onZoomIn()</slot>
  <slot>onZoomOut()</slot>
  <slot>onZoom100()</slot>
 </slots>
</ui>
